GET_STARTED.md

```diff
@@ -0,0 +1,442 @@
+
+#### Testing: Using Linux Machine
+
+To Start a test CCF network on a VM, it requires [CCF to be intalled](https://microsoft.github.io/CCF/main/build_apps/install_bin.html)
+
+Start the CCF network using the cchost in release mode
+
+```bash
+ /opt/ccf/bin/cchost --config ./config/cchost_config_enclave_js.json
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+Or virtual mode
+
+```bash
+/opt/ccf/bin/cchost --config ./config/cchost_config_virtual_js.json
+...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+Now, a network is started with one node and one member, you need to execute the following governance steps to initialize the network, [check Network governance section](#network-governance)
+
+- Activate the network existing member (to start a network governance)
+- Build the application and [create a deployment proposal](#build-application)
+- Deploy the application proposal, [using governance calls](#network-governance)
+- Create and submit [an add users proposal](#new-user-proposal)
+- Open the network for users ([using proposal](#open-network-proposal))
+
+##### CCF Node Configuration file
+
+To start or join new node you need some configs, The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+#### Testing: Using Azure Managed CCF Serivce ([MCCF](https://techcommunity.microsoft.com/t5/azure-confidential-computing/microsoft-introduces-preview-of-azure-managed-confidential/ba-p/3648986))
+
+To test you application using MCCF, you can create Azure Managed CCF Serivce on your subscription, the service will create a CCF network which is (initialized, Active member, and In Open State)
+
+- First, create the network's first member certificate, please check [Certificates generation](https://microsoft.github.io/CCF/release/3.x/governance/adding_member.htmlhttps://microsoft.github.io/CCF/release/3.x/governance/adding_member.html)
+- Create a new Azure Managed CCF Serivce (first member certificate required as input)
+- Build the application and [create a deployment proposal](#build-application)
+- Deploy the application proposal, [using governance calls](#network-governance)
+- Create and submit [an add users proposal](#new-user-proposal)
+
+### Testing: Application Endpoints
+
+Now you can send requests to your endpoints, the following sample requests to log application [(ccf-app-template)](https://github.com/microsoft/ccf-app-template/tree/main/js)
+
+In another terminal:
+
+#### Banking application
```

Comment:
At this point - would it not be better to reference a doc that lives inside the banking-app folder?

Reply:
Done, but later we need to add some testing samples to the application doc